src/com/platypub/feat/posts.clj

```diff
@@ -304,13 +324,13 @@
         [:.text-lg.my-2 "Drafts"]
         (->> drafts
              (sort-by :post/edited-by #(compare %2 %1))
-             (map post-list-item))
+             (map #(post-list-item db %)))
```

Comment:
Rather than pass the db in, I'd probably change the `(pull post [*])` above to `(pull post [* {:post/sites [*]}])`